.storybook/preview.js

```diff
@@ -2,7 +2,6 @@ import React from "react";
 import "./story-styles.css";
 import "dist/style.css";
 import { NdsStyles } from "./decorators";
-import { Layout } from "./Layouts";
```

Comment:
We no longer need this. In the future, we can declare a set of autodocs layouts available to use.